.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.{fs,fsx}]
+fsharp_multiline_block_brackets_on_same_column=true
+fsharp_multi_line_lambda_closing_newline=true
+fsharp_blank_lines_around_nested_multiline_expressions=false
+fsharp_experimental_stroustrup_style=true
+fsharp_keep_max_number_of_blank_lines=1
```

Comment:
Can we change the rule to below? Thanks!!!
```txt
[*.{fs,fsx}]
indent_size=4
max_line_length=150
fsharp_max_if_then_short_width=80
fsharp_max_if_then_else_short_width=80
fsharp_max_infix_operator_expression=120
fsharp_max_record_width=60
fsharp_max_record_number_of_items=5
fsharp_max_array_or_list_width=120
fsharp_max_array_or_list_number_of_items=100
fsharp_max_value_binding_width=120
fsharp_max_function_binding_width=120
fsharp_max_dot_get_expression_width=120
fsharp_multiline_block_brackets_on_same_column=true
fsharp_multi_line_lambda_closing_newline=true
fsharp_blank_lines_around_nested_multiline_expressions=false
fsharp_experimental_stroustrup_style=true
```

Reply:
Done, I did remove `indent_size=4` as this is the default.
I also skipped `fsharp_max_record_number_of_items` and `fsharp_max_array_or_list_number_of_items` as these settings require `fsharp_max_array_or_list_width=number_of_items` to be set as well.

See [docs](https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html#fantomas-setting-icon-type-green-fantomas-setting-icon-fsharp_max_array_or_list_number_of_items).
Do you want these enabled as well?

Reply:
Thanks, everything should be ok. I will merge it.